pkg/crawler/log.go

```diff
@@ -17,7 +20,21 @@ func (c *Crawler) safePrintln(statusCode int, message string) {
 
 	if c.statusFilter.IsValid(c.FilterStatusQuery, int64(statusCode)) {
 		c.printMutex.Lock()
+		// Console log
 		_, _ = fmt.Fprintln(c.OutWriter, message)
```

Comment:
I think when we log to a file we should not print out the messages anymore also specified in #4

Reply:
Resolved.